src/cookbooks/looker/GUD.md

```diff
@@ -0,0 +1,42 @@
+# Growth and Usage dashboards
+
+The Mozilla Growth & Usage dashboards (GUD) are visualizations of growth metrics in a standard way across Mozilla’s products.
+
+The dashboards have been migrated to Looker from the [previous GUD Dashboard](https://mozilla.github.io/gud/) as part of the Looker on-boarding in Mozilla.
```

Comment:
Just personal preference I'd put this at the end or in an appendix if the Looker dashboards are meant to be the primary thing

Reply:
Definitely, thanks.